style/middleware/_middleware.forums.scss

```diff
@@ -0,0 +1,118 @@
+/* ==========================================================================
+   #FORUMS
+   ========================================================================== */
+/**
+ * Unfortunately, Moodle provides Forums that we cannot actually get access to
+ * in order to add new classes. Here we’re just remapping its selector onto
+ * the cleaner Nightingale ones.
+ */
+
+// Add new Forum button
+.singlebutton.forumaddnew button {
+  @extend .c-btn;
+  @extend .c-btn--submit;
+  @extend .c-btn--full;
+}
+
+// Forum Search
+.forumsearch {
+  float: right;
+}
+
+.forumsearch button {
+  @extend .c-sprite;
+  @extend .c-sprite--magnifying-glass-rev;
+  float: right;
+}
+
+.forumsearch input > text {
+  @extend .c-form-input;
+}
+
+
+
+// Removing border of first post for every forum so it could stand out
+.forumpost {
+  border: none;
+}
+
+.forumpost .row .topic, .forumpost .row .content-mask, .forumpost .row .options {
+  margin-left: 0px;
+}
+
+// Hiding Avatar image from each forum post/thread as not required for v1.0
+.left.picture {
+  display: none;
+}
+
+// Making first post thread topic/subject bolder
+.topic.firstpost.starter .subject {
+  font-weight: 700;
```

Comment:
Are we extending some kind of bold / bigger typeface for this or just rolling our own new style that isn't in Nightingale?

Reply:
No, but I have still changed it to extend `strong` CSS styling from core-nightingale